pkg/schedule/schedulers/scheduler_controller.go

```diff
@@ -127,6 +139,10 @@ func (c *Controller) isSchedulingHalted() bool {
 // ResetSchedulerMetrics resets metrics of all schedulers.
 func (c *Controller) ResetSchedulerMetrics() {
 	schedulerStatusGauge.Reset()
+	ruleStatusGauge.Reset()
+	// create in map again
+	rulesCntStatusGauge = ruleStatusGauge.WithLabelValues("rule_count")
```

Comment:
Why do we use recreate?

Reply:
I wonder if the rest will clear the metricsMap.

Reply:
ptal @rleungx